internal/storage/disk/disk.go

```diff
@@ -55,7 +56,7 @@ func NewStore(ctx context.Context, conf *Conf) (*Store, error) {
 	s := &Store{
 		conf:                conf,
 		idx:                 idx,
-		log:                 zap.S().Named(DriverName),
+		log:                 zap.L().Named("disk.store").With(zap.String("dir", conf.Directory), zap.Bool("watchForChanges", conf.WatchForChanges)),
```

Comment:
We don't have to keep a pointer to the log in this struct because we don't use the logger anywhere else other than the `Reload` function. 

In the `Reload` function we can grab the logger from context. So we don't need a global logger.

Reply:
Do we need access to `conf`? As far as I can see, it is being used to set the logger keys here. But, since we are getting the logger from the context in `Reload`, they are irrelevant.

Reply:
After a private chat, turns out I misunderstood the previous review. The resolution is to use the logger coming from `ctx` since the `Reload` function of the store is directly called by a service (as opposed to other store functions working independently from API calls)